include/crm/common/mainloop.h

```diff
@@ -28,6 +28,15 @@ enum mainloop_child_flags {
     mainloop_leave_pid_group = 0x01,
 };
 
+struct mainloop_timer_s {
+        guint id;
+        guint period_ms;
+        bool repeat;
+        char *name;
+        GSourceFunc cb;
+        void *userdata;
+};
```

Comment:
Opaque objects are very useful ... instead of exposing this, let's add a getter function for period_ms. I eventually want to make pacemaker's mainloop API private, so make the getter private. That would put it somewhere like include/crm/common/internal.h and prefixed with pcmk__.